texasbbq.py

```diff
@@ -111,6 +111,9 @@ def git_checkout(tag):
 
 def conda_update_conda():
     execute("conda update -y -n base -c defaults conda")
+    # FIXME: remove this when https://github.com/conda/conda/pull/9665 is
+    # merged and released
+    execute("conda install conda=4.7")
```

Comment:
i think you need a `-y`.

Reply:
Yeah, I thought so too, but in the output I see:

```
::>> running: 'conda install conda=4.7'
Collecting package metadata (current_repodata.json): - \ | done
Solving environment: - failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): | / - \ | / - \ | / - \ | / - \ | / - \ | / - \ | done
Solving environment: - \ | / - \ | / - \ | / - \ | / - \ | / - \ | / - done

## Package Plan ##

  environment location: /home/circleci/repo/miniconda3

  added / updated specs:
    - conda=4.7


The following packages will be DOWNGRADED:

  conda                                        4.8.2-py37_0 --> 4.7.12-py37_0


Proceed ([y]/n)? 
Preparing transaction: | done
Verifying transaction: - done
Executing transaction: | done
```

Reply:
And then later on I see:

```
==> WARNING: A newer version of conda exists. <==
  current version: 4.7.12
  latest version: 4.8.2

Please update conda by running

    $ conda update -n base -c defaults conda
```

Which would indicate, that it was installed anyway. Maybe it is a bug in conda, or there is a `.condarc` file kicking about.

Reply:
It seems to work without too (and I don't know why) but I added it to be safe.